From c05458236ac591a88b8031e2e313b94bbe691beb Mon Sep 17 00:00:00 2001
From: Tri Khuong <tri.t.khuong@realsenseai.com>
Date: Fri, 31 Oct 2025 23:34:09 -0700
Subject: [PATCH] t23x: add overlays for D4xx cameras

Signed-off-by: Tri Khuong <tri.t.khuong@realsenseai.com>
---
 overlay/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/overlay/Makefile b/overlay/Makefile
index c88bbe2..37b4c96 100644
--- a/overlay/Makefile
+++ b/overlay/Makefile
@@ -60,6 +60,9 @@ dtbo-y += tegra234-p3767-camera-p3768-imx219-A.dtbo
 dtbo-y += tegra234-p3767-camera-p3768-imx219-imx477.dtbo
 dtbo-y += tegra234-p3767-camera-p3768-imx477-C.dtbo
 dtbo-y += tegra234-p3767-camera-p3768-imx477-A.dtbo
+dtbo-y += tegra234-camera-d4xx-overlay.dtbo
+dtbo-y += tegra234-camera-d4xx-overlay-dual.dtbo
+dtbo-y += tegra234-p3767-camera-p3768-d4xx-A.dtbo
 
 ifneq ($(dtb-y),)
 dtb-y := $(addprefix $(makefile-path)/,$(dtb-y))
-- 
2.34.1

